index.yaml

```diff
@@ -14567,7 +14567,7 @@ entries:
       catalog.cattle.io/upstream-version: 45.31.1
     apiVersion: v2
     appVersion: v0.65.1
-    created: "2023-11-16T14:06:07.046533148-05:00"
```

Comment:
This was a conflict and I fixed the asset created date with the one from `release-v2.8` which is the source of truth for released assets.